README.md

```diff
@@ -21,6 +21,7 @@ Configuration keys supported only by the data-table plugin:
 | :-- | :-- | :-- | :-- |
 | `entityType`  | "edge" \| "node" (**required**)  | Whether to display nodes or edges in the table. | `"node"` |
 | `itemType`    | string (**required**)            | Name of the node category or edge type to display in the table. | `"Company"` |
+| `delimiter`    | string (**optional**)           | The delimiter used for csv files downloads. By default "," is used | `";"` |
```

Comment:
additionally it would be nice to update the checkPluginConfig method called here: https://github.com/Linkurious/lke-plugin-data-table/blob/d3c7884d8deb00fa43cea174ee817e00fb0e0751/backend/routeHandler.js#L46 to check that the delimiter is 1 char long.

Reply:
Done